apps/nextjs-website/src/app/[productSlug]/[...releaseNoteSubPathSlugs]/page.tsx

```diff
@@ -33,26 +33,6 @@ type ReleaseNotePageStaticParams = {
   releaseNoteSubPathSlugs: string[];
 };
 
```

Comment:
Like we have discuss in meeting i will keep these pages static for now

```suggestion
// TODO: remove when release notes metadata will be managed in strapi
export const revalidate = REVALIDATE_LONG_INTERVAL;

const PRODUCT_SLUG_PATH_INDEX = 1;
const RELEASE_NOTE_SUB_PATH_INDEX = 2;
export async function generateStaticParams(): Promise<
  ReleaseNotePageStaticParams[]
> {
  const releaseNotes = await getReleaseNotesMetadata();
  const releaseNoteParams = releaseNotes
    .map(({ path }) => path.split('/'))
    .filter((paths) => paths.length > RELEASE_NOTE_SUB_PATH_INDEX)
    .map((paths) => {
      return {
        productSlug: paths[PRODUCT_SLUG_PATH_INDEX],
        releaseNoteSubPathSlugs: paths.slice(RELEASE_NOTE_SUB_PATH_INDEX),
      };
    });
  return releaseNoteParams;
}
```